chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -54,4 +56,9 @@ void MigrateObsoleteProfilePrefs(Profile* profile) {
   // Added 1/2020
   brave_wallet::MigrateBraveWalletPrefs(profile);
 #endif
+
+  // Added 6/2020
+  brave_shields::MigrateFingerprintingPrefsFromV1ToV2(
```

Comment:
we intentionally did not migrate settings cc @pes10k @pilgrim-brave

Reply:
Migration commit is removed from this PR.